### DISCUSSION TITLE
Adding an image best practices

### DISCUSSION THREAD

**Discussion by eshanholtz** [twilio-labs/paste#2326] [Q&A] [answered]
I'm building a UI with an error state use case where an image is positioned to the left of Header, Paragraph, and Button Paste components. Would it be best to use the Media Object component to wrap these items or should I use a Box as an image or a primitive image tag for the image?

**Comment by shleewhite** [comment, marked as answer]
Hi @eshanholtz! This is a great use case for Media Object. It helps with laying out images inline with other items, there's more info on the docs page: https://paste.twilio.design/components/media-object#about-media-object.

Here is an example of how you could build the UI: https://codesandbox.io/s/angry-wiles-ymgvj7?file=/src/App.tsx

Please let me know if you have more questions.

**Comment by eshanholtz** [comment]
Thank you!
